src/meta/MetaVersionMan.cpp

```diff
@@ -330,8 +313,13 @@ Status MetaVersionMan::doUpgrade(kvstore::KVStore* kv) {
             }
         }
     }
-    return Status::OK();
+    if (!setMetaVersionToKV(kv)) {
```

Comment:
Not only set the version when upgrade. when the meta has not set the version also need to set the version

Reply:
the 2.0 data couldn't call doUpgrade, the doUpgrade directly to decode by v1

Reply:
OK, I will move it to daemon as before.